### PR TITLE
feat: update cart quantity functionality

### PR DESCRIPTION
**Pull request by bhtibrewal**
This is a small pr with just cart quantity update functionality

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
Code looks good to me, feel free to merge

File: src/components/card/components/QuantitySection.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        className={`btn btn-primary ${qty <= 1 && "disabled-btn"}`}
+        onClick={() => {
+          if (qty > 1)
+            updateCartQuantity({
+              _id,
+              action: { type: "decrement" },
+              productDispatch,
+              showToast,
+            });
+        }}
```
Comment (pankaj-prog) [inline src/components/card/components/QuantitySection.jsx:23]: also, handle the case if qty is 1, you can show some message to the user or you could remove the item from cart.

**Review by pankaj-prog — APPROVED** [review]
- Code is clean and readable. Nice work

File: src/components/card/components/QuantitySection.jsx (hunk 1 above)
Comment (bhtibrewal) [reply src/components/card/components/QuantitySection.jsx:23]: okay thanks for the suggestion

**Pull request merged**